content/definition/definition.de.md

```diff
@@ -0,0 +1,15 @@
++++
+aliases = ["/translations/de"]
++++
+
+## Die Definition von Ethical Source
+
+Software wird als _Ethical Source_ betrachtet wenn es folgende Kriterien erfüllt:
```

Comment:
wenn _sie_ folgende Kriterien erfüllt?

Reply:
Danke Stefan! Für einfache Korrekturen kannst du das ["Suggestion" feature](https://help.github.com/en/articles/reviewing-proposed-changes-in-a-pull-request) nutzten, ist super hilfreich :)

Zum Beispiel

````
```suggestion
Software wird als _Ethical Source_ betrachtet wenn sie folgende Kriterien erfüllt:
```
````